.chloggen/add_otlp.yaml

```diff
@@ -8,7 +8,7 @@ component: awsfirehosereceiver
 note: make otlp_v1 a valid record type
 
 # Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
-issues: [35750]
+issues: [35750, 36125]
```

Comment:
I'm adding the pr number here in the existing changelog since both PRs are fixing the same issue. Is that OK? If so, can I get the "Skip Changelog" label added please? Thanks!